vyper/semantics/types/module.py

```diff
@@ -53,6 +58,15 @@ def abi_type(self) -> ABIType:
     def __repr__(self):
         return f"interface {self._id}"
 
+    def _try_fold(self, node):
+        if len(node.args) != 1:
+            raise UnfoldableNode("wrong number of args", node.args)
+        args = [arg.get_folded_value() for arg in node.args]
+        if not isinstance(args[0], vy_ast.Hex):
+            raise UnfoldableNode("not an address", node.args[0])
```

Comment:
```suggestion
        arg = node.args[0].get_folded_value()
        if not isinstance(arg, vy_ast.Hex):
            raise UnfoldableNode("not an address", arg)
```
I think this can be simplified.